tracex_project/extraction/views.py

```diff
@@ -59,16 +64,35 @@ def form_valid(self, form):
             event_types=form.cleaned_data["event_types"],
             locations=form.cleaned_data["locations"],
         )
-        orchestrator.run(view=self)
-        single_trace_df = orchestrator.data
-        single_trace_df = utils.Conversion.prepare_df_for_xes_conversion(
-            single_trace_df, orchestrator.configuration.activity_key
-        )
-        utils.Conversion.create_xes(
-            utils.CSV_OUTPUT,
+        if IS_TEST:
+            default_journey = PatientJourney.manager.get(name="default_journey_name")
+            trace = (
+                Trace.manager.filter(patient_journey=default_journey)
+                .order_by("last_modified")
+                .first()
+            )
+            print(trace)
+            event_objects = trace.events.all()
+            data = {
+                "case_id": trace.id,
+                "activity": [event.activity for event in event_objects],
+                "event_type": [event.event_type for event in event_objects],
+                "start": [event.start for event in event_objects],
+                "end": [event.end for event in event_objects],
+                "duration": [event.duration for event in event_objects],
+                "attribute_location": [event.location for event in event_objects],
+            }
+            orchestrator.data = pd.DataFrame(data)
+            orchestrator.simulate_extraction(view=self)
+        else:
+            orchestrator.run(view=self)
+
+        utils.Conversion.create_xes_from_df(
+            orchestrator.data,
             name="single_trace",
             key=orchestrator.configuration.activity_key,
         )
+        print(orchestrator.data)
```

Comment:
Do we need this `print` or is it debugging?

Reply:
oh this is an oversight on my part. definetly not needed. Nice catch!

Reply:
Removed.